controllers/osbuildenvconfig_controller_test.go

```diff
@@ -961,25 +1006,639 @@ var _ = Describe("OSBuildEnvConfig Controller", func() {
+
+																							Context("The certificate for the external builder exists", func() {
+																								var (
+																									externalBuilderCertificateName = fmt.Sprintf(workerCertificateNameFormat, externalBuilderName)
+																									externalBuilderCertificate     *certmanagerv1.Certificate
+																								)
+
+																								BeforeEach(func() {
+																									externalBuilderCertificate = &certmanagerv1.Certificate{
+																										ObjectMeta: metav1.ObjectMeta{
+																											Name:            externalBuilderCertificateName,
+																											Namespace:       operatorNamespace,
+																											OwnerReferences: []metav1.OwnerReference{ownerReference},
+																										},
+																										Spec: certmanagerv1.CertificateSpec{
+																											SecretName: externalBuilderCertificateName,
+																											PrivateKey: &certmanagerv1.CertificatePrivateKey{
+																												Algorithm: "ECDSA",
+																												Size:      256,
+																											},
+																											DNSNames: []string{
+																												externalBuilderName,
+																											},
+																											Duration: &metav1.Duration{
+																												Duration: time.Hour * certificateDuration,
+																											},
+																											IssuerRef: certmanagermetav1.ObjectReference{
+																												Group: "cert-manager.io",
+																												Kind:  "Issuer",
+																												Name:  caIssuerName,
+																											},
+																										},
+																									}
+																									certificateRepository.EXPECT().Read(requestContext, externalBuilderCertificateName, operatorNamespace).Return(externalBuilderCertificate, nil)
+																								})
+
+																								Context("The Secret of the certificate of the external worker is owned by the instance", func() {
+																									var (
+																										externalBuilderCertificateSecret *corev1.Secret
+																									)
+
+																									BeforeEach(func() {
+																										externalBuilderCertificateSecret = &corev1.Secret{
+																											ObjectMeta: metav1.ObjectMeta{
+																												Namespace:       operatorNamespace,
+																												Name:            internalBuilderCertificateName,
+																												OwnerReferences: []metav1.OwnerReference{ownerReference},
+																											},
+																										}
+																										secretRepository.EXPECT().Read(requestContext, externalBuilderCertificateName, operatorNamespace).Return(externalBuilderCertificateSecret, nil)
+																									})
+
+																									Context("The configmap for the playbook of the external builder setup exists", func() {
```

Comment:
configmap -> configMap

Reply:
Done (and I found more cases of this fix)